src/screens/Portfolio/AnimatedTopBar.js

```diff
@@ -16,14 +16,30 @@ import BalanceHeader from "./BalanceHeader";
 import HeaderErrorTitle from "../../components/HeaderErrorTitle";
 import HeaderSynchronizing from "../../components/HeaderSynchronizing";
 
-class AnimatedTopBar extends PureComponent<{
-  scrollY: AnimatedValue,
-  portfolio: Portfolio,
-  counterValueCurrency: Currency,
-  pending: boolean,
-  error: ?Error,
-  navigation: *,
-}> {
+class AnimatedTopBar extends PureComponent<
+  {
+    scrollY: AnimatedValue,
+    portfolio: Portfolio,
+    counterValueCurrency: Currency,
+    pending: boolean,
+    error: ?Error,
+    navigation: *,
+  },
+  { skipRender: boolean },
+> {
+  state = {
+    skipRender: true,
+  };
+  componentDidMount() {
+    this.props.scrollY.addListener(({ value }) =>
+      this.setState({ skipRender: value < 90 }),
```

Comment:
This can be pretty bad for performance. this triggers a setState regardless if the value changes, which means potentially a lot of redraw.

```
setState({ skipRender: false })
setState({ skipRender: false })
setState({ skipRender: false })
setState({ skipRender: true })
setState({ skipRender: true })
setState({ skipRender: true })
```

would trigger 6 re-render()


What you can do instead is this:


```
this.setState(old => {
  const skipRender = value < 90;
  if (skipRender===old.skipRender) return null;
  return { skipRender };
});
```

Reply:
nice